process.md

```diff
@@ -1,13 +1,13 @@
 # SDL Evolution Process
-The SDL maintainers and stakeholders seek the help of the community to help guide and shape how SDL changes. To do that effectively, we outline in this document a process for introducing ideas to SDL and how the SDL maintainers will guide those ideas through the review process.
+The SmartDeviceLink Consortium (SDLC) seeks the help of the community to help guide and shape how SDL changes. To do that effectively, we outline in this document a process for introducing ideas to SDL and how the SDLC Steering Committee will guide those ideas through the review process.
 
 ## Scope
-Any changes to the [SDL RPC spec](https://github.com/smartdevicelink/rpc_spec/blob/master/spec.xml), the [SDL protocol](https://github.com/smartdevicelink/protocol_spec), public API changes to the SDL [iOS](https://github.com/smartdevicelink/sdl_ios) or [Android](https://github.com/smartdevicelink/sdl_android) SDKs, or major changes to [SDL Core](https://github.com/smartdevicelink/sdl_core) must go through the SDL evolution proposal and review process. SDL is versioned using [semantic versioning](http://www.semver.org). Therefore, any public API removal, alteration, or addition is a minor or major version change to the respective spec or library, no matter how small the change. The only minor or major change that does not need to go through this process is a misspelled API name in a mobile SDK. In that case, write an issue against the respective repository. Changes that do not make a spec or public API change should go through the normal contribution process, for example, [this is the iOS repository's process](https://github.com/smartdevicelink/sdl_ios/blob/master/.github/CONTRIBUTING.md).
+Any changes to the [SDL RPC spec](https://github.com/smartdevicelink/rpc_spec), the [SDL protocol](https://github.com/smartdevicelink/protocol_spec), public API changes to the SDL [iOS](https://github.com/smartdevicelink/sdl_ios) or [Android](https://github.com/smartdevicelink/sdl_android) SDKs, or major changes to [SDL Core](https://github.com/smartdevicelink/sdl_core) must go through the SDL evolution proposal and review process. SDL is versioned using [semantic versioning](http://www.semver.org). Therefore, any public API removal, alteration, or addition is a minor or major version change to the respective spec or library, no matter how small the change. The only minor or major change that does not need to go through this process is a misspelled API name in a mobile SDK. In that case, write an issue against the respective repository. Changes that do not make a spec or public API change should go through the normal contribution process, for example, [this is the iOS repository's process](https://github.com/smartdevicelink/sdl_ios/blob/master/.github/CONTRIBUTING.md).
```

Comment:
I think we actually agreed on an enhancement / bug fix scheme over the API change scheme. Some changes may not actually change the API surface but will need proposals. It's an enhancement if it changes the way developers use the project, and a bug fix if it changes it, but changes it to the way it was supposed to work, but failed to do so.